neuromation/cli/main.py

```diff
@@ -441,6 +452,8 @@ def help(ctx: click.Context, command: Sequence[str]) -> None:
 cli.add_command(share.acl)
 
 cli.add_command(DeprecatedGroup(storage.storage, name="store", hidden=True))
+if os.environ.get("_NEURO_OBJ_STORAGE"):
```

Comment:
Depending on how we merge/release the functionality I kind of did this to hide it during development.

Reply:
In this case, I hide the commands until we release it. Don't want to halt a release if needed, as servers are yet to be provisioned with object storage. 
What would be the behaviour of CLI commands if Object Storage is not supported for this cluster? Just an error when you try to use it?